pkg/security/ebpf/c/exec.h

```diff
@@ -59,7 +59,7 @@ struct exit_event_t {
     struct process_context_t process;
     struct span_context_t span;
     struct container_context_t container;
-    u32 exit_code;
+    u64 exit_code;
```

Comment:
I'm not sure why we need to move from u32 to u64 here, it seems to me that in the eBPF code and in the userspace code we only ever look at the lower two bytes

Reply:
applied